dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/bundle/persister/RelationshipPersister.java

```diff
@@ -96,10 +96,28 @@ protected void updatePreheat( TrackerPreheat preheat, org.hisp.dhis.relationship
         // NOTHING TO DO
     }
 
+    @Override
+    protected boolean isUpdatable()
```

Comment:
it indeed appears as ignored

Reply:
But it will not appear in the stats as ignored, right?
If so, is this acceptable?

Reply:
When importing 2 existing TEIs and a relationship (existing as well), report returns 2 updated and 1 ignored. It is also asserted in e2e tests so it would be correct. However tests are just checking relationship stats, I'll double check if general stats are updated accordingly.

Reply:
👍

Reply:
If tests are ok, no need to check.
I just had this doubt, but I think that the ignored are counted by exclusion in the end, so everything should just work fine.